### PR TITLE
Update KW Channel Logos

### PR DESCRIPTION
**Pull request by boeder8**
Changed logos for 9x KW channels using the updated official logos.

KTV1.kw
KTV2.kw
KTVAlMajlis.kw
KTVAlQurain.kw
KTVArabe.kw
KTVEthraa.kw
KTVKhallikBilbait.kw (logo only rebranded to KTV Drama - alternate name added) 
KTVSport.kw
KTVSportPlus.kw (alternate name also added as per the official website/logo)

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -13177,7 +13177,7 @@ KTVEDT1.us,KTVE-DT1,,NBC,,US,US-LA,Monroe,s/US-LA,eng,,FALSE,,,,https://www.myar
 KTVEDT2.us,KTVE-DT2,,FOX,,US,US-LA,Monroe,s/US-LA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f0/Fox_Networks_Group_US_logo.svg/512px-Fox_Networks_Group_US_logo.svg.png
 KTVEDT3.us,KTVE-DT3,,Laff,,US,US-AR,El Dorado,s/US-AR,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/1/1e/LAFF_logo.png
 KTVEDT4.us,KTVE-DT4,,Ion Mystery,,US,US-AR,El Dorado,s/US-AR,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/fb/ION_Mystery_logo.svg/512px-ION_Mystery_logo.svg.png
-KTVEthraa.kw,KTV Ethraa,,,,KW,,,c/KW,ara,,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/7uzAvQU.png
+KTVEthraa.kw,KTV Ethraa,,,,KW,,,c/KW,ara,,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/uOZOdqL.png
```
Comment (freearhey) [inline data/channels.csv:13180]: Logo is too large (https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels).

```suggestion
KTVEthraa.kw,KTV Ethraa,,,,KW,,,c/KW,ara,,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/hqEbc49.png
```

File: data/channels.csv [hunk 2]
```diff
@@ -13229,8 +13229,8 @@ KTVRDT4.us,KTVR-DT4,,opbmusic,,US,US-OR,La Grande,s/US-OR,eng,,FALSE,,,,,https:/
 KTVSLD1.us,KTVS-LD1,,,,US,US-NM,Albuquerque,s/US-NM,eng,,FALSE,,,,,https://i.imgur.com/ECTfuRA.png
 KTVSLD2.us,KTVS-LD2,,Almavision,,US,US-NM,Albuquerque,s/US-NM,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/c/c0/Almavision.PNG
 KTVSLD3.us,KTVS-LD3,,The Grio,,US,US-NM,Albuquerque,s/US-NM,eng,,FALSE,,,,,https://i.imgur.com/GKRbUyX.png
-KTVSport.kw,KTV Sport,,,,KW,,,c/KW,eng;ara,sports,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/CKw9kwZ.png
-KTVSportPlus.kw,KTV Sport Plus,,,,KW,,,c/KW,ara,sports,FALSE,,,,,https://i.imgur.com/h3KYxEp.png
+KTVSport.kw,KTV Sport,,,,KW,,,c/KW,eng;ara,sports,FALSE,,,,https://media.gov.kw/,https://i.imgur.com/R1hGX1d.png
+KTVSportPlus.kw,KTV Sport Plus,,KTV Sport 2,,KW,,,c/KW,ara,sports,FALSE,,,,,https://i.imgur.com/l4oX0gf.png
```
Comment (freearhey) [inline data/channels.csv:13233]: Alternative channel names must be specified in the third column (https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels)

```suggestion
KTVSportPlus.kw,KTV Sport Plus,KTV Sport 2,,,KW,,,c/KW,ara,sports,FALSE,,,,,https://i.imgur.com/l4oX0gf.png
```

File: data/channels.csv [hunk 3]
```diff
@@ -13197,7 +13197,7 @@ KTVKDT1.us,KTVK-DT1,,,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,http://www.azfamil
 KTVKDT2.us,KTVK-DT2,,Comet TV,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/9/97/Comet_logo.svg/512px-Comet_logo.svg.png
 KTVKDT3.us,KTVK-DT3,,Circle,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Circle_Network_logo.svg/512px-Circle_Network_logo.svg.png
 KTVKDT4.us,KTVK-DT4,,THIS,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/This_TV_logo.svg/512px-This_TV_logo.svg.png
-KTVKhallikBilbait.kw,KTV Khallik Bilbait,,,,KW,,,c/KW,ara,,FALSE,,,,,https://i.imgur.com/nykv5J5.png
+KTVKhallikBilbait.kw,KTV Khallik Bilbait,,KTV Drama,,KW,,,c/KW,ara,,FALSE,,,,,https://i.imgur.com/krojJJ7.png
```
Comment (freearhey) [inline data/channels.csv:13200]: Alternative channel names must be specified in the third column (https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels)

```suggestion
KTVKhallikBilbait.kw,KTV Khallik Bilbait,KTV Drama,,,KW,,,c/KW,ara,,FALSE,,,,,https://i.imgur.com/krojJJ7.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 1 above)
Comment (boeder8) [reply data/channels.csv:13180]: can't commit suggestion as I get the error: "This diff has recently been updated."

File: data/channels.csv (hunk 3 above)
Comment (boeder8) [reply data/channels.csv:13200]: made a new commit with all the changes requested

**Review by freearhey — APPROVED** [review]

**Pull request merged**